src/Microsoft.ComponentDetection.Contracts/TypedComponent/VcpkgComponent.cs

```diff
@@ -71,4 +81,22 @@ public override PackageURL PackageUrl
             }
         }
     }
+
+    private void SetGitRepoProperties()
+    {
+        /* example download locations
+         * "git+https://github.com/leethomason/tinyxml2@9.0.0"
+         * "git+https://github.com/Microsoft/vcpkg#ports/nlohmann-json"
+        */
+        var locationArr = this.DownloadLocation.Split('/');
```

Comment:
qq, if downloadlocation is null will the split work?

Reply:
I have a null or empty check before this method gets called